shared/macros-bash.jinja

```diff
@@ -285,11 +285,11 @@ fi
 
 {{%- macro _make_bash_variable_assignment(varname, value="", quoted_value="") -%}}
 {{% if value -%}}
-	{{{ varname }}}="{{{ value }}}"
+    {{{ varname }}}="{{{ value }}}"
 {{%- elif quoted_value -%}}
-	{{{ varname }}}="\"{{{ quoted_value }}}\""
+    {{{ varname }}}="\"{{{ quoted_value }}}\""
 {{%- else %}}
-	{{{ raise("Specify either 'value' or 'quoted_value' as macro arguments.") }}}
+    {{{ raise("Specify either 'value' or 'quoted_value' as macro arguments.") }}}
```

Comment:
Next time, please don't do changes in unrelated macros even if it's just white space or typo fix.